rls/src/main/java/io/grpc/rls/RlsRequestFactory.java

```diff
@@ -73,11 +73,11 @@
   RouteLookupRequest create(String service, String method, Metadata metadata) {
     checkNotNull(service, "service");
     checkNotNull(method, "method");
-    String path = service + "/" + method;
+    String path = "/" + service + "/" + method;
     Map<String, NameMatcher> keyBuilder = keyBuilderTable.row(path);
     // if no matching keyBuilder found, fall back to wildcard match (ServiceName/*)
     if (keyBuilder.isEmpty()) {
-      keyBuilder = keyBuilderTable.row(service + "/*");
+      keyBuilder = keyBuilderTable.row("/" + service + "/*");
```

Comment:
Also need update javadoc of `keyBuilderTable`.

Reply:
good catch. done.